src/Controllers/MovieController.cs

```diff
@@ -55,12 +60,18 @@ public IActionResult EditMovie(int id)
                 movieToBeChanged.Update(movie);
                 UnitOfWork.Movies.Update(movieToBeChanged);
                 UnitOfWork.Complete();
-                return RedirectToAction("EditMovie");
+                return RedirectToAction("ListMovies");
             }
             catch(NullReferenceException n){
                 return NotFound();
             }
         }
+
+        [HttpGet("ListMovies")]
+        public IActionResult ListMovies(){
+            IEnumerable movieList = UnitOfWork.Movies.GetAll();
```

Comment:
Try using the MovieViewModel list instead of the Movie list (you should be able to convert them easily using LINQ)